services/tinlake/actions.ts

```diff
@@ -95,7 +95,16 @@ export async function issue(tinlake: any, tokenId: string, nftRegistryAddress: s
   };
 }
 
-export async function getLoan(tinlake: any, loanId: string): Promise<TinlakeResult> {
+export async function getProxyOwner(tinlake: any, loanId: string) : Promise<TinlakeResult> {
+  let owner = ZERO_ADDRESS;
+  try {
+    owner = await tinlake.getProxyOwnerByLoan(loanId);
+  } catch (e) {
+  }
```

Comment:
Should we not handle this error somehow? At least console.log for debugging? Or do we use sentry/bugsnag or other loggers we can send it to?

Reply:
we are not using a third party error logger.
We were logging this error before, but closed loans don't have owners, since the nfts get burned. So this function call would always throw and spam the console 🙈